components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3525,4 +3531,25 @@ void RewardsServiceImpl::OnWalletCreatedForSetAdsEnabled(
   }
 }
 
+std::string RewardsServiceImpl::GetExternalWalletType() const {
+  const int32_t current_country =
+      country_codes::GetCountryIDFromPrefs(profile_->GetPrefs());
+
+  // TODO(zenparsing): [BF POST-MVP] Rename |kOnlyAnonWalletCountries| to
+  // reflect current usage
+  for (const auto& country : kOnlyAnonWalletCountries) {
+    if (country.length() == 2) {
+      const int id = country_codes::CountryCharsToCountryID(
+          country.at(0), country.at(1));
+
+      if (id == current_country)
+        return ledger::constant::kWalletBitflyer;
+    }
+  }
+
+  // TODO(zenparsing): For development, we just want to use bitflyer everywhere
+  return ledger::constant::kWalletBitflyer;
+  // return ledger::constant::kWalletUphold;
```

Comment:
?

Reply:
I think this may have been sent for review accidentally. This is just a private branch with some work that's not meant for `master` yet.

Reply:
This was @justnom rebasing this branch in b-c when it should have only been rebased in b-b from where it pins the feature branch. I've erased history on this.